packages/@react-spectrum/tabs/test/Tabs.test.js

```diff
@@ -670,4 +670,51 @@ describe('Tabs', function () {
     expect(onSelectionChange).toBeCalledTimes(1);
     expect(onSelectionChange).toHaveBeenCalledWith(items[0].name);
   });
+
+  it.only('updates the tab index of the selected tab if programatically changed', function () {
+    let {getAllByRole, rerender} = render(
+      <Provider theme={theme}>
+        <Tabs aria-label="Test Tabs" items={items} selectedKey={'Tab 3'}>
+          <TabList>
+            {item => (
+              <Item key={item.name} title={item.name || item.children} />
+            )}
+          </TabList>
+          <TabPanels>
+            {item => (
+              <Item key={item.name}>
+                {item.children}
+              </Item>
+            )}
+          </TabPanels>
+        </Tabs>
+      </Provider>
+    );
+
+    let tabs = getAllByRole('tab');
+    expect(tabs[2]).toHaveAttribute('tabindex', '0');
```

Comment:
may as well check all of them, technically in a roving tab index all items are tabindex 0 until something has been focus has entered for the first time